pkg/daemon/daemon.go

```diff
@@ -1716,8 +1694,27 @@ func (dn *Daemon) prepUpdateFromCluster() (*mcfgv1.MachineConfig, *mcfgv1.Machin
 // "transient state" file, which signifies that all of those prior steps have
 // been completed.
 func (dn *Daemon) completeUpdate(desiredConfigName string) error {
-	if err := dn.cordonOrUncordonNode(false); err != nil {
-		return err
+	if err := dn.nodeWriter.SetDesiredDrainer(dn.kubeClient.CoreV1().Nodes(), dn.nodeLister, dn.name, fmt.Sprintf("%s-%s", "uncordon", desiredConfigName)); err != nil {
+		return fmt.Errorf("Could not set drain annotation: %w", err)
+	}
+
```

Comment:
Lets add a call to `dn.startConfigDriftMonitor()` here. This will ensure that the Config Drift Monitor is active before we uncordon the node.